botocore/session.py

```diff
@@ -1030,6 +1037,17 @@ def _resolve_defaults_mode(self, client_config, config_store):
 
         return lmode
 
+    def _add_configured_endpoint_provider(self, client_name, config_store):
+        chain = ConfiguredEndpointProvider(
+            full_config=self.full_config,
+            scoped_config=self.get_scoped_config(),
+            client_name=client_name,
+        )
+        config_store.set_config_provider(
+            logical_name='configured_endpoint_url',
```

Comment:
Is there a particular reason we are not using `endpoint_url` here as the config store key? We use the `endpoint_url` naming convention everywhere else for the parameter name to override the endpoint url so it probably makes sense to stay consistent with that name in the `config_store`.

Reply:
A vestige of previous refactoring from `session.py`, will rename.